examples/client/vendor/satellizer.js

```diff
@@ -274,8 +274,9 @@
         };
 
         shared.setToken = function(response, isLinking) {
-          var token = response.access_token || config.tokenRoot && response.data[config.tokenRoot] ?
-            response.data[config.tokenRoot][config.tokenName] : response.data[config.tokenName];
+          var token;
```

Comment:
Actually I have no idea how it got show up here. But I made the changes from the master branch.
This change was previously in commit https://github.com/sahat/satellizer/commit/937b2bccd9cf9bbd6530d1886f0a06f40f17a3c2 from the repo.